src/bin/test_window.c

```diff
@@ -13,12 +13,19 @@ int main() {
     struct graphics_color bg = {0,0,0};
     struct event e;
     int x = 0;
+    int count = 0;
     while (1) {
         int res = get_event(&e);
         if (res == 0) {
+            if (count > 10) {
+                clear();
+                count = 0;
+                x = 0;
+            }
             if (e.type == EVENT_KEYBOARD_PRESS) {
                 draw_char(x, 0, e.character, &c, &bg);
```

Comment:
Right now we can draw directly onto the border, and when we clear we won't clear the bits that were drawn. Do we want the border to be part of the window? This is relatively minor, we don't need to fix it now.

<img width="752" alt="screen shot 2016-05-03 at 16 41 40" src="https://cloud.githubusercontent.com/assets/4718844/14997714/2b34e7bc-114e-11e6-8acd-8e2b1b0e29cf.png">